sphinxcontrib/confluencebuilder/builder.py

```diff
@@ -70,6 +72,16 @@ def init(self, suppress_conf_check=False):
         if not ConfluenceConfig.validate(self.config, not suppress_conf_check):
             raise ConfluenceConfigurationError('configuration error')
 
+        if self.config.confluence_ask_password:
+            print('(request to accept password from interactive session)')
+            print(' Instance: ' + self.config.confluence_server_url)
+            print('     User: ' + self.config.confluence_server_user)
+            sys.stdout.write(' Password: ')
+            sys.stdout.flush()
+            self.config.confluence_server_pass = getpass('')
```

Comment:
Are these three lines working around a bug in getpass()? If so, how about moving this to a helper method with proper documentation before the next guy just "fixes" it to `getpass('Password:')`